nixos/modules/services/monitoring/tuptime.nix

```diff
@@ -0,0 +1,84 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.tuptime;
+
+in {
+
+  options.services.tuptime = {
+
+    enable = mkEnableOption "the total uptime service";
+
+    timer = {
+      enable = mkOption {
```

Comment:
`mkEnableOption`

Reply:
that defaults to `false`, this option should be `true` by default

Reply:
Gosh ignore me... too early 😳